modules/ztp-image-based-upgrade-with-backup.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-with-backup_{context}"]
+= Upgrading the {sno} clusters with {lcao}
+
+With the generated seed image and after completing the `Prep` stage, you can start upgrading your cluster.
+During the upgrade process, the OADP Operator creates a backup of the cluster, then {lcao} starts reimaging the target cluster.
+
+[IMPORTANT]
+====
+If the upgrade fails or stops, you must initiate a rollback. For more information, see next section.
+In this case, all the resources created by the {lcao} are automatically deleted.
+====
+
+.Prerequisites
+
+// Anything different from the prep stage?
+
+.Procedure
+
+. Move to the upgrade stage by changing the `Prep` value to `Upgrade` in the `ImageBasedUpgrade` CR.
++
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1alpha1
+kind: ImageBasedUpgrade
+metadata:
+  name: example-upgrade
+spec:
+  stage: Upgrade
+  seedImageRef:
+    version: 4.14.0
+    image: quay.io/<seed_image_repository>
```

Comment:
We're pointing to a container image here, and it doesn't necessairly have to be on quay.io

```suggestion
    image: <seed_container_image>
```